<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.0.1 (Python 3.12.5 on win32)" generated="2025-03-14T08:18:51.574159" rpa="false" schemaversion="5">
<suite id="s1" name="Robot Interactive Console" source="c:\Users\leoda.PC-LEO\.vscode\extensions\robocorp.robotframework-lsp-1.13.0\src\robotframework_ls\vendored\robotframework_interactive\robot_interactive_console.robot">
<test id="s1-t1" name="Default Task/Test" line="5">
<kw name="Interpreter Main Loop" owner="MainLoop">
<status status="PASS" start="2025-03-14T08:18:51.608158" elapsed="685.061688"/>
</kw>
<status status="PASS" start="2025-03-14T08:18:51.608158" elapsed="685.062689"/>
</test>
<status status="PASS" start="2025-03-14T08:18:51.576158" elapsed="685.095688"/>
</suite>
<statistics>
<total>
<stat pass="1" fail="0" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="1" fail="0" skip="0" id="s1" name="Robot Interactive Console">Robot Interactive Console</stat>
</suite>
</statistics>
<errors>
<msg time="2025-03-14T08:18:51.568158" level="WARN">Error in file 'c:\Users\leoda.PC-LEO\.vscode\extensions\robocorp.robotframework-lsp-1.13.0\src\robotframework_ls\vendored\robotframework_interactive\robot_interactive_console.robot' on line 4: Singular section headers like '*** Test Case ***' are deprecated. Use plural format like '*** Test Cases ***' instead.</msg>
</errors>
</robot>
